src/openrct2-ui/WindowManager.cpp

```diff
@@ -994,7 +994,7 @@ class WindowManager final : public IWindowManager
     /**
      * Closes all windows with the specified window class.
      *  rct2: 0x006ECCF4
-     * @param cls (cl) with bit 15 set
+     * @param WindowClass enum
```

Comment:
Documenting the implementation is a bit odd if you ask me, if the plan is to actually document functions like this then it should be done in the interface as that is what the user will face when used. Also we are not exactly designing a library for general purpose use so I would also argue that those comments should be brief and only if the function is relatively complex in design.

Reply:
I think we should just scrap most of them, especially on functions like this where the name pretty much states what it does and by using a modern IDE you should be able to tell what sort of parameter it expects without having to read the code.

Reply:
I would be fine with scrapping them altogether as well. @Gymnasiast, your thoughts?

Reply:
I am fine either way. I can see Matt’s point and I generally try to avoid comments that just state what was already obvious, and I think the function names here already make it abundantly clear what they do.

Reply:
Well in this case I'm in favor of scrapping them and also look out for any of those in the future, they are also quite inconsistent. Might worth to open a new issue for someone to review comments and either delete or correct them.